### ISSUE TITLE
Coordinate flip

### ISSUE THREAD

**Issue by benmaier** [ulfaslak/network_styling_with_d3#16]
When exporting from the visualization, the y-positions are flipped, such that

```
newy = height - y
```

This is done so we can have a coordinate system which looks like this

```
y
^
|
|
|-----------> x
```

In the js-vis, however, we have coordinate system which looks like this

```
|-----------> x
|
|
|
v
y
```

We have to keep that in mind, when positions written to a graph object (to node attributes 'x' and 'y') and then passed to the visualizations. Maybe we should just flip them back when the graph is loaded in js.

**Comment by benmaier** [comment]
Also, the zoom value has to be regarded when loading networks with pre-given coordinates. Maybe first load the zoomscaler, then load x and y values but immediately convert them using the inverse function of the zoomscaler

**Comment by ulfaslak** [comment]
Finally addressed this in latest commit (8ed465120f465b91b7bb7c6256992ecd0635c3b3). Now, data is coming out of the visualization like: 

|-----------> x
|
|
|
v
y

The point is that when inputting the `network_properties` to `visualize` again (enabled in recent commit on netwulf), we don't have to do any coordinate flipping.

Simultaneously, I flipped the coordinate plotting in `netwulf.tools.draw_netwulf`.

**Comment by benmaier** [comment]
okay, cool!
